components/homeComponents/HackCountdown.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { useState, useEffect, CSSProperties } from 'react';
+
+interface Hack24CountdownProps {
```

Comment:
The typename should be changed to something else since this is generic hackportal.